puppeteer/latest.Dockerfile

```
@@ -91,22 +92,25 @@ RUN npm install -g n;
 RUN pip install trcli
 
 # Install sbt
-RUN curl -L -o /root/sbt.zip https://github.com/sbt/sbt/releases/download/v1.2.8/sbt-1.2.8.zip \
+# https://github.com/sbt/sbt/releases
+RUN curl -L -o /root/sbt.zip https://github.com/sbt/sbt/releases/download/v1.10.7/sbt-1.10.7.zip \
```

Hadolint (2.12.0)

[warning] Either use Wget or Curl but not both

(DL4001)